.github/workflows/domain_ci.yml

```
@@ -56,7 +56,7 @@ jobs:
           path: vision
 
       - name: Run torchvision builtin datasets tests
-        run: pytest --no-header -v vision/test/test_prototype_builtin_datasets.py
+        run: pytest --no-header -v vision/test/test_prototype_datasets*.py
```

You probably want

```suggestion
        run: pytest --no-header -v vision/test/test_prototype_datasets_builtin.py
```

Otherwise you will also run our utility tests and the like that are unrelated to `torchdata`.

---

Or if you actually want to run all tests, you need to use 

```suggestion
        shell: bash
        run: pytest --no-header -v vision/test/test_prototype_datasets*.py
```

to be able to use the `*.py` globbing

---

Sounds great. Thank you for the suggestion.